backend/unitTests/chatAttackTest.py

```
@@ -0,0 +1,243 @@
+        print(f"Prompt Injection Status: {response.status_code}")
+        print(f"Prompt Injection Response: {response.json()}")
+        
+        if response.status_code == 200:
+            result = response.json()
+            response_text = result.get("response", "").lower()
+            # Check if response contains data that shouldn't be accessible
+            if "password" in response_text or "admin" in response_text:
+                print("🚨 SECURITY BREACH: Prompt injection successful - accessed restricted data!")
+                return True
+            return False
+        return False
```

Ruff (0.16.1)

[warning] Consider moving this statement to an `else` block

(TRY300)